openapi/components/requestBodies/SubscriptionItemUpdate.yaml

```
@@ -0,0 +1,12 @@
+content:
+  application/json:
+    schema:
+      type: object
+      required:
+        - quantityUsed
+      properties:
+        quantityUsed:
```

I wonder if it should be `quantityUsed` or `usedQuantity`?
```suggestion
        quantityUsed:
          x-badge: Experimental
```
Even though the badge may not display without some tweaks, we should note it.